manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+            if azimuth_direction is None
+            else azimuth_direction
+        )
+        super().__init__(
+            color=color,
+            x_line_frequency=self.azimuth_frequency * TAU,
+            y_line_frequency=radius_line_frequency,
+            x_axis_config=radius_config,
+            x_min=None if (radius_max is None) else -radius_max,
+            x_max=radius_max,
+            y_min=-config["frame_x_radius"] if (radius_max is None) else -radius_max,
+            y_max=config["frame_x_radius"] if (radius_max is None) else radius_max,
+            **kwargs,
+        )
+        self.azimuth_buff = azimuth_buff
+        self.prepare_for_nonlinear_transform()
+        self.background_lines.apply_function(
+            lambda p: np.array([p[0] * np.sin(p[1]), p[0] * np.cos(p[1]), 0])
+        )
+
+    def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        for mob in self.background_lines.family_members_with_points():
+            num_curves = mob.get_num_curves()
+            if num_inserted_curves > num_curves:
+                mob.insert_n_curves(num_inserted_curves - num_curves)
+        return self
+
+    def polar_to_point(self, radius, azimuth):
+        """Gets a point from polar coordinates.
+
+        Parameters
+        ----------
+        radius : :class:`float`
+            The coordinate radius (:math:`r`).
+
+        azimuth : :class:`float`
+            The coordinate azimuth (:math:`\\theta`).
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+            The point.
+        """
+        return self.coords_to_point(radius * np.cos(azimuth), radius * np.sin(azimuth))
+
+    def pr2pt(self, radius, azimuth):
+        """Abbreviation for `polar_to_point`"""
+        return self.polar_to_point(radius, azimuth)
+
+    def point_to_polar(self, point):
+        """Gets polar coordinates from a point.
+
+        Parameters
+        ----------
+        point : :class:`numpy.ndarray`
+            The point.
+
+        Returns
+        -------
+        Tuple[:class:`float`, :class:`float`]
+            The coordinate radius (:math:`r`) and the coordinate azimuth (:math:`\\theta`).
+        """
+        x, y = self.point_to_coords(point)
+        return np.sqrt(x ** 2 + y ** 2), np.arctan2(y, x)
+
+    def pt2pr(self, point):
+        """Abbreviation for `point_to_polar`"""
+        return self.point_to_polar(point)
+
+    def get_coordinate_labels(self, r_vals, a_vals, **kwargs):
+        if r_vals is None:
+            r_vals = [
+                r for r in self.get_x_axis().default_numbers_to_display() if r >= 0
+            ]
+        if a_vals is None:
+            a_vals = np.arange(0, 1, self.azimuth_frequency)
+        r_mobs = self.get_x_axis().get_number_mobjects(*r_vals)
+        if self.azimuth_direction == "CCW":
+            d = 1
+        elif self.azimuth_direction == "CW":
+            d = -1
+        else:
+            d = 0
+            ValueError()
```

Comment:
?

Reply:
The `ValueError()` is there to catch any string other than "CW" or "CCW". I'll be the first to admit I'm not well versed in pythonic conventions, so it's very possible there's a more correct way of doing this.